torch/csrc/distributed/rpc/script_call.h

```diff
@@ -27,6 +27,8 @@ class TORCH_API ScriptCall {
   Message toMessage();
   static ScriptCall fromMessage(const Message& message);
 
+  virtual ~ScriptCall() = default;
+
```

Comment:
What is the significance here?

Perhaps the subclasses were overriding this without the virtual kwarg? I recall this not even being possible (as in, you have to have a virtual dtor on a base class if you have any virtual functions), but could be wrong.

Reply:
This may cause a memory leak. If you have virtual functions it should have virtual destructor.